src/Provider/AbstractProvider.php

```diff
@@ -31,7 +31,9 @@
 
     public $responseType = 'json';
 
-    public $headers = null;
+    public $headers = [];
+
+    public $authorizationHeader;
```

Comment:
Will existing third-party providers need to update themselves to support this addition?